lib/cusolver/libcusolver.jl

```diff
@@ -1610,7 +1610,7 @@ end
     @ccall libcusolver.cusolverDnXtrtri_bufferSize(handle::cusolverDnHandle_t,
                                                    uplo::cublasFillMode_t,
                                                    diag::cublasDiagType_t, n::Int64,
-                                                   dataTypeA::cudaDataType, A::Ptr{Cvoid},
+                                                   dataTypeA::cudaDataType, A::CuPtr{Cvoid},
```

Comment:
Can you update these changes in https://github.com/JuliaGPU/CUDA.jl/blob/master/res/wrap/cusolver.toml?

Reply:
Tim, the routines with the wrong argument types are not in res/wrap/cusolver.toml.
How do you (re)generate this toml file?

Reply:
`cusolver.tomlm` is not generated, it's a human-curated list of definitions, and the changes that need to be made to the argument types. When we parse the C headers, we can only see pointers, and not the difference between CPU and GPU pointers, so we cannot autogenerate this file.

Reply:
Thanks :+1: 
I updated `cusolver.toml`.